src/main/java/uk/gov/hmcts/probate/model/DocumentType.java

```diff
@@ -68,6 +68,9 @@
     @JsonProperty("caveatRaised")
     CAVEAT_RAISED("caveatRaised"),
 
+    @JsonProperty("grantRaised")
+    GRANT_RAISED("grantRaised"),
```

Comment:
This document type needs to be added to probate-commons!

Reply:
Adding do not merge as the new document type ("grantRaised") needs to be added to commons / orchestrator first?

Reply:
Is this part resolved now?

Reply:
So I checked this and it doesn't need to be part of probate-commons. The Grant types in Probate Commons only get used for new Grant cases however when calling this notification endpoint it will be using Grant cases defined in back office instead.